src/libcore/macros.rs

```diff
@@ -231,8 +231,8 @@ macro_rules! try {
 
 /// Calls `write_fmt` function on a writer
```

Comment:
@steveklabnik Hello. Can you suggest a better, "friendlier" description instead of this? 😄